src/config.ts

```diff
@@ -53,7 +53,11 @@ const config = {
   localStorageJWTKey: 'persistJWT',
   applicantDeadlineCode: 'applicantDeadline',
   isProductionBuild,
-  debounceTimeout: 350, // milliseconds
+  debounceTimeout: 350, // milliseconds,
+  // These are the only default tables in the system that we allow to be mutated
+  // directly by modifyRecord or display as data views. All other names must
+  // have "data_table_" prepended.
+  dataViewAllowedTableNames: ['user', 'organisation', 'file'],
```

Comment:
This was previously only relevant to the back-end, but now we need it so the Data View editor knows what tables to offer. i.e -- all of the tables registered in `data_table` table, plus these three. (The back-end also includes "application" in its allowed tables, but I didn't think we need that here, since we already have a comprehensive application list)